Moderation_Matrix.md

```diff
@@ -0,0 +1,50 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
+  - Drug related conversation (Rule 7)
+  - Special attention to treating everyone with respect
+  - Mental health issues discussion (Rule 12)
+- A second infraction of the following:
+  - Short circuiting a teaching moment (Rule 10)
+- Asking for help with Windows or encouraging the use of it
+  - “Chat bombing”, aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) (Rule 7)
+
+## 2 Zaps (WARNING)
+Medium severity violations such as: 
+- Suggesting piracy (Rule 9)
+- Unsolicited pings/DM/friend request (Rule 6)
+- Political or religious topics (Rule 8)
+- Mild toxicity (directed insults, provoking/baiting other users, passive aggression, naming & shaming) (Rule 7)
+- Self promo without permission (ask with Modmail)(Rule 7)
+  - Do not share resources you have created solely for personal or monetary gain
+  - Examples: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (Could mislead learners), "Why PHY is superior" (unrelated blog post)
+
+## 5 Zaps
+High severity violations such as: 
+- Posting about any illegal activities, such as facilitating piracy (Rule 7 and 9)
+- Publicly arguing about receiving moderation activity after being asked to take it to ModMail/discuss privately 
+- Excessive toxicity (targeted harassment, inciting drama, aggressively arguing purposefully, etc.) (Rule 7)
+
+## 10 Zaps (Immediate Ban)
+Extreme severity violations such as: 
+- Bigotry (racism, homophobia, hate speech etc) (Rule 7 and 8)
+- Harassment (Rule 7)
+- NSFW Imagery (Rule 7)
+- Spamming 
+  - Discord Nitro spam
+  - Self promotion (joining just to promote material, no prior chat history)
+  - Repeated random nonsensical messages (copypasta)
+- Doxxing (Rule 7)
+
```

Comment:
I feel like we will probably want to throw a catch-all under here for when people do something not explicitly outlined but is deserving of a ban. Along the lines of "Any other behavior deemed unacceptable and deserving of a ban by the moderation team".